src/client/routes/index.js

```diff
@@ -48,21 +48,22 @@ export default (store) => {
       </Route>
 
       <Route onEnter={onProjectEnter}>
-        <Route path="/project/:id" component={Project}>
```

Comment:
Any reason why the name is not part of the top `/project/:name` instead of adding these `**/` and spats?